dwitter/tests/templatetags/test_insert_magic_links.py

```diff
@@ -189,6 +189,14 @@ def test_insert_magic_single_character_hashtag(self):
             insert_magic_links('#1')
         )
 
+    def test_insert_magic_link_anchor_not_hashtag(self):
+        self.assertEqual(
+            '<a href="/h/start">#start</a> '
+            'https://www.site.com/page#anchor '
+            '<a href="/h/end">#end</a>',
+            insert_magic_links('#start https://www.site.com/page#anchor #end')
```

Comment:
To be pedantic, https://tools.ietf.org/html/rfc6761:

```suggestion
            'https://www.example.com/page#anchor '
            '<a href="/h/end">#end</a>',
            insert_magic_links('#start https://www.example.com/page#anchor #end')
```

Reply:
Didn't know that, updated!